authors/apps/profiles/serializers.py

```diff
@@ -0,0 +1,16 @@
+from rest_framework import serializers
+
+from .models import Profile
+
+
+class ProfileSerializer(serializers.ModelSerializer):
+    username = serializers.CharField(source='user.username')
+    bio = serializers.CharField(allow_blank=True, required=False)
+    # image = serializers.SerializerMethodField()
```

Comment:
@ruganda do we need this ☝️ ?

Reply:
@arthuroe , 
- Yes we need it for retrieving a user's profile while not authenticated, or when another authenticated user is retrieving another user's profile . `api/profiles/<username>`
- `api/user` , `retrieve` and `update` can be accessed by the owner of the profile

Reply:
I have removed the comment. Sorry about the miss communication